wrp/header_wrp.go

```diff
@@ -9,17 +9,23 @@ import (
 
 // Constant HTTP header strings representing WRP fields
 const (
-	MsgTypeHeader        = "X-Midt-Msg-Type"
-	ContentTypeHeader    = "X-Midt-Content-Type"
-	AcceptHeader         = "X-Midt-Accept"
-	TrasactionUuidHeader = "X-Midt-Transaction-Uuid"
-	StatusHeader         = "X-Midt-Status"
-	RDRHeader            = "X-Midt-Request-Delivery-Reponse"
-	HeadersArrHeader     = "X-Midt-Headers"
-	IncludeSpansHeader   = "X-Midt-Include-Spans"
-	SpansHeader          = "X-Midt-Spans"
-	PathHeader           = "X-Midt-Path"
-	SourceHeader         = "X-Midt-Source"
+	MsgTypeHeader         = "X-Midt-Msg-Type"
+	ContentTypeHeader     = "X-Midt-Content-Type"
+	AcceptHeader          = "X-Midt-Accept"
+	TransactionUuidHeader = "X-Midt-Transaction-Uuid"
+	StatusHeader          = "X-Midt-Status"
+	RDRHeader             = "X-Midt-Request-Delivery-Reponse"
+	HeadersArrHeader      = "X-Midt-Headers"
+	IncludeSpansHeader    = "X-Midt-Include-Spans"
+	SpansHeader           = "X-Midt-Spans"
+	PathHeader            = "X-Midt-Path"
+	SourceHeader          = "X-Midt-Source"
+
+	ErrInvalidMsgType         = "Invalid Message Type"
```

Comment:
Each of these should be constants of type `error`:

    ErrInvalidMsgType = errors.New("Invalid Message Type")